README.md

```diff
@@ -1,57 +1,122 @@
 
 [![License: GPL-3](https://img.shields.io/badge/License-GPL3-yellow.svg)](https://opensource.org/licenses/gpl-3.0.html)
 
 ## :speech_balloon: Introduction
 
+The module consists of alignment processing steps, such as alignment of `.fastq`-files. and duplicates marking
+`.bam`-files.
+
 ## :heavy_exclamation_mark: Dependencies
 
-To run this workflow, the following tools need to be available:
+In order to use this module, the following dependencies are required:
 
-[![python](https://img.shields.io/badge/python-3.8-blue)
-[![snakemake](https://img.shields.io/badge/snakemake-6.8.0-blue)](https://snakemake.readthedocs.io/en/stable/)
-[![singularity](https://img.shields.io/badge/singularity-3.7-blue)](https://sylabs.io/docs/)
+[![hydra-genetics](https://img.shields.io/badge/hydragenetics-v0.9.1-blue)](https://github.com/hydra-genetics/)
+[![pandas](https://img.shields.io/badge/pandas-1.3.1-blue)](https://pandas.pydata.org/)
+[![python](https://img.shields.io/badge/python-3.8-blue)](https://www.python.org/)
+[![snakemake](https://img.shields.io/badge/snakemake-6.10.0-blue)](https://snakemake.readthedocs.io/en/stable/)
+[![singularity](https://img.shields.io/badge/singularity-3.0.0-blue)](https://sylabs.io/docs/)
 
 ## :school_satchel: Preparations
 
-### Sample data
-
-1. Add all sample ids to `samples.tsv` in the column `sample`.
-2. Add all sample data information to `units.tsv`. Each row represents a `fastq` file pair with
-corresponding forward and reverse reads. Also indicate the sample id, run id and lane number, adapter.
+### Sample and unit data
+
+Input data should be added to [`samples.tsv`](https://github.com/hydra-genetics/prealignment/blob/develop/config/samples.tsv)
+and [`units.tsv`](https://github.com/hydra-genetics/prealignment/blob/develop/config/units.tsv).
+The following information need to be added to these files:
+
+| Column Id | Description |
+| --- | --- |
+| **`samples.tsv`** |
+| sample | unique sample/patient id, one per row |
+| **`units.tsv`** |
+| sample | same sample/patient id as in `samples.tsv` |
+| type | data type identifier (one letter), can be one of **T**umor, **N**ormal, **R**NA |
+| platform | type of sequencing platform, e.g. `NovaSeq` |
+| machine | specific machine id, e.g. NovaSeq instruments have `@Axxxxx` |
+| flowcell | identifer of flowcell used |
+| lane | flowcell lane number |
+| barcode | sequence library barcode/index, connect forward and reverse indices by `+`, e.g. `ATGC+ATGC` |
+| fastq1/2 | absolute path to forward and reverse reads |
+| adapter | adapter sequences to be trimmed, separated by comma |
 
 ### Reference data
 
-1. You need a ...
+You need have a indexed reference genome: ex reference.fna 
+
+For bwa the files are generated by [bwa index](http://bio-bwa.sourceforge.net/bwa.shtml#:~:text=SYNOPSIS-,bwa%20index%20ref.fa,-bwa%20mem%20ref):
+| File | Description |
+| -----| ----------- |
+| reference.dict | dictionary file|
```

Comment:
This one comes from `picard CreateSequenceDictionary`, right?

Reply:
done

Reply:
fixed